lighthouse-extension/app/src/popup.js

```diff
@@ -172,6 +172,11 @@ document.addEventListener('DOMContentLoaded', _ => {
         }
       }
 
+      if (err.code === 'PAGE_LOAD_ERROR') {
```

Comment:
sorry to add one last thing, but it would be great to fold this into the known error messages handling above. Error matching is on `message` string, not `code`, but since it appears we never actually expose the message directly, it would be easy to prefix `mainRecord.localizedFailDescription` with some  string (`'Unable to load the page: '` or whatever), then use that as the key in `NON_BUG_ERROR_MESSAGES`